foundry.toml

```diff
@@ -2,6 +2,7 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
+fs_permissions = [{ access = "read", path = "./"}]
```

Comment:
Would you mind changing the permissions to be more granular?

I think that it's enough to have it like `fs_permissions = [{ access = "read", path = "./artifacts"}, { access = "read", path = "./node_modules/@uniswap"}]`